tests/TestCase.php

```diff
@@ -219,6 +219,12 @@ protected function refreshTestDatabase()
         $this->beginDatabaseTransactionOnConnection($this->landlordDatabaseConnectionName());
     }
 
+    /**
+     *
+     * @param array<string> $paths
+     *
+     * @return string
```

Comment:
This `@return` is not needed since it adds no additional details over the real return type